staging/publishing/rules.yaml

```diff
@@ -2265,6 +2265,8 @@ rules:
       branch: master
     - repository: component-base
       branch: master
+    - repository: kms
```

Comment:
No, it seems to be new. Apparently it's now referenced indirectly through CEL - at least that is when it showed up.

`k8s.io/dynamic-resource-allocation/structured/namedresources/cel` doesn't absolutely have to be in staging, but I think it's better if it is because I foresee a need by driver developers to test their CEL expressions.